src/ai/backend/agent/agent.py

```diff
@@ -1345,6 +1347,7 @@ async def lifecycle_task_exception_handler(
         ) as tg:
             while True:
                 ev = await self.container_lifecycle_queue.get()
+                log.info("process_lifecycle_events(): received lifecycle event: {!r}", ev)
```

Comment:
It would be better to log only the type values instead of logging the event itself.